[2023-06-01T11:20:50.548+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.delete_data_from_table scheduled__2022-02-19T00:00:00+00:00 [queued]>
[2023-06-01T11:20:50.667+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.delete_data_from_table scheduled__2022-02-19T00:00:00+00:00 [queued]>
[2023-06-01T11:20:50.672+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-06-01T11:20:50.673+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 6
[2023-06-01T11:20:50.675+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-06-01T11:20:50.770+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2022-02-19 00:00:00+00:00
[2023-06-01T11:20:50.796+0000] {standard_task_runner.py:55} INFO - Started process 958 to run task
[2023-06-01T11:20:50.834+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'delete_data_from_table', 'scheduled__2022-02-19T00:00:00+00:00', '--job-id', '227', '--raw', '--subdir', 'DAGS_FOLDER/postgres.py', '--cfg-path', '/tmp/tmpq1liycrj']
[2023-06-01T11:20:50.846+0000] {standard_task_runner.py:83} INFO - Job 227: Subtask delete_data_from_table
[2023-06-01T11:20:51.503+0000] {task_command.py:389} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.delete_data_from_table scheduled__2022-02-19T00:00:00+00:00 [running]> on host 687ba427e841
[2023-06-01T11:20:51.989+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=dag_with_postgres_operator_v03
AIRFLOW_CTX_TASK_ID=delete_data_from_table
AIRFLOW_CTX_EXECUTION_DATE=2022-02-19T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-19T00:00:00+00:00
[2023-06-01T11:20:51.997+0000] {sql.py:253} INFO - Executing: 
            delete from dag_runs where dt = '2022-02-19' and dag_id = 'dag_with_postgres_operator_v03';
        
[2023-06-01T11:20:52.037+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-01T11:20:53.803+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-01T11:20:53.832+0000] {sql.py:364} INFO - Running statement: 
            delete from dag_runs where dt = '2022-02-19' and dag_id = 'dag_with_postgres_operator_v03';
        , parameters: None
[2023-06-01T11:20:53.841+0000] {sql.py:373} INFO - Rows affected: 0
[2023-06-01T11:20:53.876+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=delete_data_from_table, execution_date=20220219T000000, start_date=20230601T112050, end_date=20230601T112053
[2023-06-01T11:20:53.966+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-06-01T11:20:54.049+0000] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
